profiles/default/linux/m68k/17.0/musl/make.defaults

```diff
@@ -0,0 +1,16 @@
+# Copyright 2022-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+CHOST="m68k-unknown-linux-musl"
+CHOST_m68k="${CHOST}"
+
+ARCH="m68k"
+ACCEPT_KEYWORDS="m68k ~m68k"
+
+ABI="m68k"
+DEFAULT_ABI="m68k"
+MULTILIB_ABIS="m68k"
+LIBDIR_m68k="lib"
+
+# Disable sandbox on this architecture
+FEATURES="-sandbox"
```

Comment:
Why? It works fine for me. Not under qemu-user, but that's a special case. Also if the reason sandbox doesn't work is due to seccomp (think so, can't remember), then that was implemented for m68k [very recently](https://github.com/seccomp/libseccomp/pull/397).

Reply:
I shamelessly stole it from [here](https://github.com/gentoo/gentoo/blob/master/profiles/arch/m68k/make.defaults#L23)  the logic behind it being I'll work with what I knew worked then test and remove afterwards. With this information though I'll remove it now.